### PR TITLE
Add Grafana visualizations folder and histograms doc

### PR DESCRIPTION
**Pull request by mathisve**
# Description
- Add Grafana Visualizations folder
- Add Grafana Visualizations index.md
- Updated page-index.js
- Add Grafana Histograms documentation

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    WHEN (dv_max - lag(dv_max, 1) OVER (PARTITION BY symbol ORDER BY time)) < 0 THEN dv_max
+    ELSE (dv_max - lag(dv_max, 1) OVER (PARTITION BY symbol ORDER BY time)) END vol
+FROM buckets;
+```
+This query results in the following histogram:
+<img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/volume_distribution.png" alt="volume distribution"/>
+As you can see, we get a left-skewed distribution for the AMZN symbol. In many cases there will
+be outliers that represent a few very large volume transactions that distort the distribution. One
+solution to this problem is to only look at transactions with a volume less than a certain threshold.
+But that is outside the scope of this example.
```
Comment (ryanbooz) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:196]: For now, let's either cut this sentence (just end it above) OR:

- Show an updated query with a filter based on `vol` (so, probably a having clause) - I don't think this means more screenshots. Just helpful info.
- Mention that one solution would be to use a logarithmic scale but unfortunately this chart doesn't yet support that.

Comment (mathisve) [reply timescaledb/tutorials/grafana/visualizations/histograms.md]: I used the latter in my latest commit ([2dade79](https://github.com/timescale/docs/pull/1081/commits/2dade79f64b405ee543babff11f2c816efebc096)).
Unless you meant for me to use both, then I can add another query with the filter 😄!

**Review by ryanbooz — COMMENTED** [review]
Sorry - I thought I submitted these suggestions earlier. Should be quick (honestly a little hard to tell if they still apply because GitHub is being wonky on me tonight)

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  performance issues.
+* Pre-bucketed data: This format contains data that has been pre-bucketed by the
+  data source. The Grafana documentation refers to features built into Elastic
+  Search's histogram bucket aggregation, or Prometheus' histogram metric as
+  examples. The Grafana documentation states that any data source can output
+  pre-bucketed data as long as it meets the data format requirements.
+* Aggregated data: Grafana also accepts aggregated time bucket data. Such data
+  can be easily queried using Timescale's `time_bucket()` hyperfunction. However, this also works with the `date_trunc` postgres function.
+  Grafana will automatically select a bucket size (~10% of the total range of
+  data) and bucket the aggregated data appropriately.
```
[the remark this answers is not in the document]
Comment (mathisve) [reply timescaledb/tutorials/grafana/visualizations/histograms.md]: @Loquacity got rid of these, I figured it was for stylistic reasons. I'll add them back

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+4.  When we visualize this query, we get the following histogram:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/multiple_histograms.png" alt="multiple histograms"/>
+
+    We can clearly see the 3 distinct histograms but it's impossible to tell 
+    them apart from each other. 
+    
+5.  Click on the green line left of the legend and pick a color:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/color_options.png" alt="color options"/>
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:207]:
```suggestion
    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/color_options.png" alt="Screenshot of the Grafana color picker."/>
```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+1.  We use this query to fetch all companies in our dataset:
+
+    ```sql
+    SELECT DISTINCT symbol FROM company ORDER BY asc;
+    ```
+
+2.  Create a new dashboard variable with the previous query: 
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/symbol_variable_query.png" alt="symbol variable query"/>
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:182]:
```suggestion
1.  Create a new dashboard variable with the previous query: 

        <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/symbol_variable_query.png" alt="Screenshot of Grafana showing the Variables > Edit dialog. The variable name is 'symbol', and its type is 'Query'. The Query input field has been filled out with the query from Step 1."/>
```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+6.  Doing so results in the following panel:
+    
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/colored_in_histogram.png" alt="colored in histogram"/>
+
+    Here we can clearly see the 3 different price distributions for each stock.
+
+</procedure>
+
+## Price/Volume Histogram
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:217]:
```suggestion
## Create a price/volume histogram
```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+  will automatically select a bucket size (~10% of the total range of data) and 
+  bucket the aggregated data appropriately.
+
+<highlight type="note">
+Histograms are great for analyzing the spread or distribution of data, but they
+don't show the change of data over time. If you need to see the distribution of your data over time, try a
+[heatmap](https://grafana.com/docs/grafana/latest/visualizations/heatmap/)
+instead.
+</highlight>
+
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:37]:
```suggestion

## What you'll learn
This tutorial shows you how to:
*   [Create a price/transaction histogram from raw data](#create-a-price-transaction-histogram-with-raw-data)
*   [Create a price/transaction histogram from pre-aggregated data](#create-a-price-transaction-histogram-with-aggregated-data)
*   [Create a panel showing multiple histograms](#create-a-panel-with-multiple-price-transaction-histograms)
*   [Create a price/volume histogram](#create-a-price-volume-histogram)
```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/colored_in_histogram.png" alt="colored in histogram"/>
+
+    Here we can clearly see the 3 different price distributions for each stock.
+
+</procedure>
+
+## Price/Volume Histogram
+
+An interesting metric to investigate is the trade volume distribution. This 
+distribution gives us insight into the volume at which people buy a stock.
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:220]:
```suggestion
Besides transaction price, you can also look at trade volumes. The distribution of
trade volume shows you how often and how much people are buying a stock.

```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+distribution gives us insight into the volume at which people buy a stock. 
+The dataset has a column with the daily cumulative volume which we can use to 
+determine the individual transaction volume. To calculate the volume of data for
+each bucket, we need to find the maximum `day_volume` value for the symbol and 
+then subtract each value from the previous bucket maximum value. The difference 
+of these two rows produces the volume within the bucket of time. We can 
+accomplish this with a slightly more complex query that pre-aggregates our data 
+using the TimescaleDB `time_bucket()` hyperfunction and the standard PostgreSQL 
+`MAX()` function, and then use the PostgresSQL `LAG()` window function to 
+subtract each row from the previous, ordered by `time` descending.
```
Comment (charislam) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:229]:
```suggestion
The `stocks_real_time` hypertable contains a column with the daily cumulative
volume. You can use this to calculate the volume of data for each bucket. First,
find the maximum `day_volume` value for a symbol within a bucket. Then subtract
each maximum from the previous bucket's maximum. The difference equals the
volume for that bucket.

You can do this with a pre-aggregation query, using:
*   TimescaleDB's `time_bucket` function.
*   PostgreSQL's `max` function.
*   PostgreSQL's `lag` function. Use this to subtract each from from the previous,
    when the rows are ordered by descending `time`.
```
Is descending correct here?

Also suggest linking to API references for these functions

**Pull request merged**